ui/packages/consul-ui/app/components/topology-metrics/down-lines/index.hbs

```diff
@@ -90,6 +90,13 @@
       @item={{item}}
       @oncreate={{action @oncreate item @service}}
     />
+  {{else if (and item.Intention.Allowed (not item.TransparentProxy) (eq item.Source 'specific-intention'))}}
```

Comment:
This is only in the down-lines because only downstreams would have "not defined upstreams explicitly."